bowler/main.py

```diff
@@ -58,8 +58,8 @@ def do(interactive: bool, query: str, paths: List[str]) -> None:
             }
             IPython.start_ipython(argv=[], user_ns=namespace)
 
-        except ImportError:
-            pass
+        except ImportError as e:
+            raise click.ClickException(f"failed to import IPython: {e}") from e
 
         finally:
```

Comment:
Looks like we need to drop this `finally` block as well, and move the return into the try block above, otherwise we still swallow the exception.